cmd/csi-resizer/main.go

```diff
@@ -112,15 +125,16 @@ func main() {
 		config, err = rest.InClusterConfig()
 	}
 	if err != nil {
-		klog.Fatal(err.Error())
+		klog.ErrorS(err, "Failed to create cluster config")
```

Comment:
Exit is missing here, `klog.Fatal` would exit but `ErrorS` does not.

Reply:
I've add `klog.FlushAndExit`. Could you please review it again?